Game/world/world.cpp

```diff
@@ -791,6 +795,20 @@ void World::addSound(const ZenLoad::zCVobData& vob) {
     }
   }
 
+bool World::addItemOrNpcBySymbolName (const std::string& name, const Tempest::Vec3& at) {
+    size_t id = script().getSymbolIndex(name);
+
+    if( addNpc(id, at) != nullptr ) {
```

Comment:
Type of symbol can be deducted by `script().getSymbol(id).instance.instanceOf()`

Reply:
Nice, I will change that.

Reply:
Does not work as "insert ice_dragon" will return instance Daedalus::IC_None.
Is this intended behavior?